src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -129,6 +137,38 @@ function createModel(program: Program): any {
     if (authentication) {
         auth = processServiceAuthentication(authentication);
     }
+    const consumes = getConsumes(program, serviceNamespaceType);
+    let contentTypeParameter = undefined;
+    const requestMediaTypes: string[] = [];
+    if (consumes && consumes.length > 0) {
+        contentTypeParameter = {
+            Name: "content_type",
+            NameInRequest: "content_type",
+            Type: {
+                Name: "String",
+                Kind: InputTypeKind.String,
+                IsNullable: false
+            } as InputPrimitiveType,
+            Location: RequestLocation.Header,
+            IsApiVersion: false,
+            IsResourceParameter: false,
+            IsContentType: true,
+            IsRequired: true,
+            IsEndpoint: false,
+            SkipUrlEncoding: false,
+            Explode: false,
+            Kind: InputOperationParameterKind.Method,
```

Comment:
This one should be `InputOperationParameterKind.Constant`

Reply:
Hello @AlexanderSher Those parameters are used in method signature, they should be InputOperationParameterKind.Method. What is 'InputOperationParameterKind.Constant' for?

Reply:
This is for pseudo-parameters that are not part of the signature, but instead we use their default value in request:
https://github.com/Azure/autorest.csharp/blob/8f0859eebff16a9f28cc6c6b5ea926dbb1153e03/src/AutoRest.CSharp/LowLevel/Output/OperationMethodChainBuilder.cs#L384

Reply:
Thanks @AlexanderSher I updated it.